docs/interacting-with-fleet-manager.md

```diff
@@ -0,0 +1,102 @@
+  ```
+  curl -H "Authorization: Bearer <result-of-ocm-token-command>" http://127.0.0.1:/8000/api/kafkas_mgmt
+  ```
+  OCM tokens other than the OCM offline token have an expiration time so a
+  new one will need to be generated when that happens
+
+There are some additional steps needed if you want to be able to perform
+certain actions that have additional requirements. See the
+[_User Account & Organization Setup_](getting-credentials-and-accounts.md#user-account--organization-setup) for more information
+
+### Creating a Kafka Request
+
+The following example shows how to create a Kafka Request in AWS:
+```
+curl -v -XPOST -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas?async=true -d '{ "region": "us-east-1", "cloud_provider": "aws",  "name": "test-kafka", "multi_az":true}'
+```
+
+#### Listing a Kafka Request
+
+The following example shows how to List a Kafka Request:
+```
+curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_request_id> | jq
+```
+
+### Listing all Kafka Requests
+The following example shows how to List all Kafka Requests:
+```
+curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas | jq
+```
+
+### Deleting a Kafka Request
+The following example shows how to delete a Kafka Request
+```
+curl -v -X DELETE -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_request_id>?async=true
+```
+
+### Using the Kafka Admin Server API
+
+The Kafka Admin Server API is used for managing topics, acls, and consumer groups
+on a given Kafka Request.
+
+The Kafka Admin Server OpenAPI specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/src/main/resources/openapi-specs/kafka-admin-rest.yaml).
+
+To get the Kakfa Admin Server API endpoint for a given Kafka instance: after the
+Kafka instance is in ready state call the GET kafka Instance endpoint against
+the Fleet Manager API.
+
+Assuming the Fleet Manager is running in a local process on port 8000 that
+can be done by executing:
+```
+curl -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_id> | jq .admin_api_server_url
+```
+## Interacting using the RHOAS CLI
+
+An alternative to interact with the Fleet Manager API is to make use of the
+[RHOAS CLI](https://github.com/redhat-developer/app-services-cli).
+
+First [run the Fleet Manager locally](../README.md#running-fleet-manager-for-the-first-time-in-your-local-environment)
+
+Then, run the RHOAS CLI pointing to the locally running Fleet Manager:
+```
+./rhoas login --mas-auth-url=stage --api-gateway=http://localhost:8000
+```
+
+> Note: The locally installed kas-fleet-manager doesn't deploy TLS enabled
+kafka admin server but the default URL scheme used by the app-service-cli is
+HTTPS. So, the url scheme should be changed to http in the
```

Comment:
It should be possible to configure the locally KFM to have TLS enabled for the admin server. 
There are the `enable-cname-registration` and `enable-kafka-external-listener` flags.

Reply:
:+1: Note deleted.